src/nix/hive/options.nix

```diff
@@ -282,6 +282,16 @@ with builtins; rec {
         default = {};
         type = types.attrsOf types.unspecified;
       };
+      disallowApplyAll = lib.mkOption {
```

Comment:
I think we should change it to `allowApplyAll` just to be consistent with `allowLocalDeployment`.